modules/cloudbuild/cloudbuild_builder/Dockerfile

```diff
@@ -14,15 +14,18 @@
 
 FROM gcr.io/cloud-builders/gcloud-slim
 
-ENV TERRAFORM_VERSION=0.12.13
-ENV TERRAFORM_VERSION_SHA256SUM=63f765a3f83987b67b046a9c31acff1ec9ee618990d0eab4db34eca6c0d861ec
+ARG TERRAFORM_VERSION=0.12.24
+ARG TERRAFORM_VERSION_SHA256SUM=602d2529aafdaa0f605c06adb7c72cfb585d8aa19b3f4d8d189b42589e27bf11
+
+ENV ENV_TERRAFORM_VERSION=$TERRAFORM_VERSION
```

Comment:
Since you're using --build-arg to build, I don't think we need the `ENV` variables. We can just keep `ARG`.